overlay/buildroot/msd/img/version.js

```diff
@@ -0,0 +1,290 @@
+function OnTheWeb() {
+	window.location.assign("https://wiki.analog.com/university/tools/pluto");
+}
+
+function versionCompare(v1, v2) {
+	var v1parts = ("" + v1).replace(/[a-zA-Z]/g, "").replace("-", ".").split("."),
+	v2parts = ("" + v2).replace(/[a-zA-Z]/g, "").replace("-", ".").split("."),
+	minLength = Math.min(v1parts.length, v2parts.length),
+	p1, p2, i;
+	console.log(v1parts + " || " + v2parts);
+	for(i = 0; i < minLength; i++) {
+		p1 = parseInt(v1parts[i], 10);
+		p2 = parseInt(v2parts[i], 10);
+		if (isNaN(p1)){ p1 = v1parts[i]; }
+		if (isNaN(p2)){ p2 = v2parts[i]; }
+		if (p1 == p2) {
+			continue;
+		}else if (p1 > p2) {
+			return 1;
+		}else if (p1 < p2) {
+			return -1;
+		}
+		return NaN;
+	}
+	if (v1parts.length === v2parts.length) {
+		return 0;
+	}
+	return (v1parts.length < v2parts.length) ? -1 : 1;
+}
+
+var latest_libiio;
+var mac = false;
+
+function GetDriverurl() {
+	mac = navigator.platform.match(/Mac/i) ? true : false;
+	if (mac) {
+		var uAgent = navigator.userAgent.toLowerCase();
+		//uAgent = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.90 Safari/537.36".toLowerCase();
+		var test = uAgent.match(/mac os x 10_[0-9]*/);
+		if (test) {
+			mac = "darwin-" + test[0].match(/10_[0-9]*/)[0].replace("_", ".");
+		}
+		document.getElementById('prompt0').innerHTML = "adi-mm:tests analogdevices$";
+		document.getElementById('prompt1').innerHTML = "adi-mm:tests analogdevices$";
+	} else {
+		document.getElementById('hidehorndis').style.display = "none";
```

Comment:
My Browser (Firefox 82) encounters the following error:
```
Uncaught TypeError: document.getElementById(...) is null
    GetDriverurl http://192.168.2.1/img/version.js:46
    CheckFrmVersion http://192.168.2.1/img/version.js:218
```

Reply:
Fixed this by re-adding the section about libiio to the html file (makes sense as well, as we currently still support iio operation as well)